src/games/strategy/triplea/ui/NotesPanel.java

```diff
@@ -1,20 +1,15 @@
 package games.strategy.triplea.ui;
 
-import java.awt.Component;
 import java.awt.Rectangle;
 
 import javax.swing.BoxLayout;
-import javax.swing.JButton;
 import javax.swing.JEditorPane;
-import javax.swing.JLabel;
 import javax.swing.JPanel;
 import javax.swing.JScrollPane;
-import javax.swing.SwingUtilities;
 
 public class NotesPanel extends JPanel {
   private static final long serialVersionUID = 2746643868463714526L;
   protected final JEditorPane m_gameNotesPane;
-  final JButton m_refresh = new JButton("Refresh Notes");
```

Comment:
why remove the refresh button? AFAIK map makers use it to see how the notes will look after they update.